velox/common/memory/MmapAllocator.cpp

```diff
@@ -486,7 +486,7 @@ bool isAllZero(xsimd::batch<uint64_t> bits) {
 } // namespace
 
 int32_t MmapAllocator::SizeClass::findMappedFreeGroup() {
-  constexpr int32_t kWidth = 4;
+  constexpr int32_t kWidth = SIMD_WORD_COUNT;
```

Comment:
Can we use `xsimd::batch<int64_t>::size` here?

Reply:
Yep, done.